release.nix

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
```

Comment:
```
# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
#
# SPDX-License-Identifier: MPL-2.0
{ pkgs ? import <nixpkgs> { }, timestamp ? "19700101" }:
let closures = builtins.attrValues (import ./. { inherit pkgs timestamp; });
in pkgs.runCommandNoCC "release" { inherit closures; } ''
  mkdir -p $out
  for closure in $closures; do
    cp $closure/* $out/
  done
''
```
or alternatively if `hub release` will resolve symlinks
```
# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
#
# SPDX-License-Identifier: MPL-2.0
{ pkgs ? import <nixpkgs> { }, timestamp ? "19700101" }:
pkgs.symlinkJoin {
  name = "release";
  paths = builtins.attrValues (import ./. { inherit pkgs timestamp; });
}
```
are both simpler, more elegant and easier to maintain.